gzip/README.md

```diff
@@ -0,0 +1,31 @@
+# gzip 
+
+Build the [gzip](https://www.gnu.org/software/gzip/) data compressor as a
+static PIE ELF running on Linux.
+
+## Building
+
+The `gzip` static PIE ELF file is located in the current directory.
+If you want to rebuild it, run:
+
+```
+$ ./build.sh
+```
+
+This script downloads, unpacks, patches, configures and builds the `gzip`
+static PIE ELF file.
+
+## Running
+
+The resulting `gzip` binary can be used as is:
+
+```
+$ ./gzip <file-to-compress>
+```
+
+It is also loadable using the 
```

Comment:
don't mention the unikraft loader here, just the example on how to run it natively.